### ISSUE TITLE
error: Database Connection Error: Knex: Timeout acquiring a connection with Node 14.0.0

### ISSUE THREAD

**Issue by Frankstar** [Requarks/wiki#1766]
got the same/similiar error than in #833
OS: Centos7
Node: 14.0.0

```
2020-04-24T12:52:47.552Z [MASTER] info: =======================================
2020-04-24T12:52:47.556Z [MASTER] info: = Wiki.js 2.3.51 ======================
2020-04-24T12:52:47.556Z [MASTER] info: =======================================
2020-04-24T12:52:47.557Z [MASTER] info: Initializing...
2020-04-24T12:52:48.182Z [MASTER] info: Using database driver pg for postgres [ OK ]
2020-04-24T12:52:48.190Z [MASTER] info: Connecting to database...
2020-04-24T12:53:18.202Z [MASTER] error: Database Connection Error: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
2020-04-24T12:53:18.203Z [MASTER] warn: Will retry in 3 seconds... [Attempt 1 of 10]
2020-04-24T12:53:21.208Z [MASTER] info: Connecting to database...
```

Upgraded from `2.2.51`
`2.2.51` was running on node `13.1.0`
Needed to upgrade to node `14.0.0` for `2.3.51` cause i got the error

```
Error: Package exports for '/opt/nodejsapps/wikijs2/node_modules/nanoid' do not define a valid '.' target
```

with node `13.1.0`

mayb thats related?

**Comment by Frankstar** [comment]
looks like it IS related to the node version `14.0.0` (current stable node)
installed `13.13.0` and its working with that without the issue.

I dont know what todo to now with the issues.
Pls decide yourself @NGPixel

**Comment by NGPixel** [comment]
This is a bug in dependency node-postgres, which is fixed in version 8.0.3. I upgraded the dependency in 19dddc85c3ad5d1a6fa4ae8c74ce151968fb339e, which should fix the issue.
